transcom/mymove
# Add model/migration and handlers for form1299

## Description:
This PR adds the model, migration, and model tests for Form1299. 

## Testing Steps:

1. Run the migration: `make db_dev_migrate`.
2. Verify the table is created in the database by connecting (`bin/psql-dev`), checking that it's there (`\dt`), and looking at the column value types and options (`\d form1299s;`) :

<img width="580" alt="screen shot 2018-01-30 at 10 12 12 am" src="https://user-images.githubusercontent.com/8170735/35584083-c0526ac2-05a8-11e8-82a9-885e2ddd6323.png">

OR, 

3. Verify tests pass or the server runs: `make server_test` || `make server_run`
